jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -302,7 +319,9 @@ private static DiagnosticProperties doHelp(String diagnosticCommand) {
 	@SuppressWarnings("nls")
 	private static final String DIAGNOSTICS_GC_CLASS_HISTOGRAM_HELP = "Obtain heap information about a Java process%n"
 			+ FORMAT_PREFIX + DIAGNOSTICS_GC_CLASS_HISTOGRAM + " [options]%n"
-			+ " Options: -all : include all objects, including dead objects%n"
+			+ " Options:%n"
+			+ "          all : include all objects, including dead objects (this is default option)%n"
```

Comment:
should be "(this is *the* default option)"

Reply:
Fixed.